static/schemas/config_schema.json

```diff
@@ -126,12 +149,36 @@
       },
       "$ref": "#/definitions/AdminOptions"
     },
-    "tokio_runtime": {
+    "bifrost": {
```

Comment:
Is Bifrost an internal term or will we also use this towards users?

Reply:
This has not been decided yet. Atm, it is called bifrost in the configuration and that's why the generated specification is like this. Atm, the users won't have to configure it.